official/mnist/mnist_tpu.py

```diff
@@ -168,6 +186,19 @@ def main(argv):
   if FLAGS.eval_steps:
     estimator.evaluate(input_fn=eval_input_fn, steps=FLAGS.eval_steps)
 
+  # Run prediction on the test data.
+  if FLAGS.enable_predict:
+    predictions = estimator.predict(input_fn=predict_input_fn)
+
+    for pred_dict in predictions:
+      template = ('Prediction is "{}" ({:.1f}%), expected "{}"')
+
+      class_id = pred_dict['class_ids']
+      probability = pred_dict['probabilities'][class_id]
+      expected_label = pred_dict['label']
+
+      print(template.format(class_id, 100 * probability, expected_label))
```

Comment:
I'd just inline the template string here.

Reply:
I would like to keep it this way for better readability.